backend/doc/live_message_setup.md

```diff
@@ -17,7 +17,7 @@ you need to set tracim config like this to use default pushpin port:
 
 ~~~ini
 basic_setup.website_base_url = http://localhost:7999
-live_messages.control_uri = http://localhost:5561
+live_messages.control_zmq_uri = tcp://localhost:5563
```

Comment:
Would it be possible to use a UNIX socket (maybe more efficient than local tcp?)
Probably far from being a bottleneck though.

Reply:
maybe, but this involves modifying pushpin's configuration (as it uses tcp by default) so I propose to leave this value for now.

Reply:
:+1: